scripts/updates/utils/termux_pkg_upgrade_version.sh

```diff
@@ -69,7 +69,7 @@ termux_pkg_upgrade_version() {
 	echo "INFO: package being updated to ${LATEST_VERSION}."
 
 	sed \
-		-e "s/^\(TERMUX_PKG_VERSION=\)\(.*\)\$/\1\"${EPOCH}${LATEST_VERSION}\"/g" \
+		-e "s/^\(TERMUX_PKG_VERSION=\(?\)\(.*\)\$/\1\"${EPOCH}${LATEST_VERSION}\"/g" \
```

Comment:
`scripts/updates/utils/termux_pkg_upgrade_version.sh` doesn't currently "support" version arrays.
So this works by adding a stupid little work around to make it work with the first line of an array.
I'll borrow some code from `lint-packages` when I add array support "properly".
https://github.com/termux/termux-packages/blob/89853e9a6cc78c3694150e0b89237a8afe3a02e9/scripts/lint-packages.sh#L73-L120